crates/cli/src/subcommands/generate/mod.rs

```diff
@@ -1,11 +1,11 @@
 #![warn(clippy::uninlined_format_args)]
 
+use anyhow::Context;
```

Comment:
I don't really know how to review this file. Do you feel confident that the behavior is identical and/or that the test coverage covers all the cases that this might affect?